es_mapping.json

```diff
@@ -0,0 +1,45 @@
+{"properties": {
+				"url": {
+					"type": "text"
+				},
+				"zoneName": {
+					"type": "keyword"
+				},
+				"absolutePath": {
+					"type": "keyword"
+				},
+				"fileName": {
+					"type": "text"
+				},
+				"parentPath": {
+					"type": "text"
+				},
+				"isFile": {
+					"type": "boolean"
+				},
+				"dataSize": {
+					"type": "long"
+				},
+				"mimeType": {
+					"type": "keyword"
+				},
+				"lastModifiedDate": {
+					"type": "date",
+					"format": "epoch_millis"
```

Comment:
should just be epoch seconds?

Reply:
That caught my eye as well.  I googled it and it's an ElasticSearch-specific schema format, apparently.  Probably just means it's  a long integer ... will poke around further

Reply:
now using `epoch_second` instead of `epoch_millis`